export default function foo () {
	try {
		return 42;
	} catch ( err ) {
		console.log( err );
	}
}
